src/lib/skilling/skills/herblore/mixables/potions.ts

```diff
@@ -0,0 +1,457 @@
+	{
+		name: 'Stamina potion (4)',
+		aliases: ['stamina potion (4)', 'stamina potion', 'stamina'],
+		id: itemID('Stamina potion (4)'),
+		level: 77,
+		xp: 102,
+		inputItems: resolveNameBank({ 'Super energy (4)': 1, 'Amylase crystal': 4 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.17
+	},
+	{
+		name: 'Zamorak brew (3)',
+		aliases: ['zamorak brew (3)', 'zamorak brew'],
+		id: itemID('Zamorak brew (3)'),
+		level: 78,
+		xp: 175,
+		inputItems: resolveNameBank({ 'Torstol potion (unf)': 1, Jangerberries: 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Antidote++ (4)',
+		aliases: ['antidote++ (4)', 'antidote++'],
+		id: itemID('Antidote++ (4)'),
+		level: 79,
+		xp: 177.5,
+		inputItems: resolveNameBank({
+			'Coconut milk': 1,
+			'Irit leaf': 1,
+			'Magic roots': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.45
+	},
+	{
+		name: 'Bastion potion (3)',
+		aliases: ['bastion potion (3)', 'bastion potion', 'bastion'],
+		id: itemID('Bastion potion (3)'),
+		level: 80,
+		xp: 155,
+		inputItems: resolveNameBank({
+			'Cadantine blood potion (unf)': 1,
+			'Wine of zamorak': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Battlemage potion (3)',
+		aliases: ['battlemage potion (3)', 'battlemage potion', 'battlemage'],
+		id: itemID('Battlemage potion (3)'),
+		level: 80,
+		xp: 155,
+		inputItems: resolveNameBank({
+			'Cadantine blood potion (unf)': 1,
+			'Potato cactus': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Saradomin brew (3)',
+		aliases: ['saradomin brew (3)', 'saradomin brew', 'sara brew'],
+		id: itemID('Saradomin brew (3)'),
+		level: 81,
+		xp: 180,
+		inputItems: resolveNameBank({
+			'Toadflax potion (unf)': 1,
+			'Crushed nest': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Weapon poison(++)',
+		aliases: ['weapon poison(++)'],
+		id: itemID('Weapon poison(++)'),
+		level: 82,
+		xp: 165,
+		inputItems: resolveNameBank({
+			'Coconut milk': 1,
+			'Cave nightshade': 1,
+			'Poison ivy berries': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Extended antifire (4)',
+		aliases: ['extended antifire (4)', 'extended antifire'],
+		id: itemID('Extended antifire (4)'),
+		level: 84,
+		xp: 110,
+		inputItems: resolveNameBank({ 'Antifire potion(4)': 1, 'Lava scale shard': 4 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.17
+	},
+	{
+		name: 'Anti-venom (4)',
+		aliases: ['anti-venom (4)', 'anti-venom'],
+		id: itemID('Anti-venom (4)'),
+		level: 87,
+		xp: 120,
+		inputItems: resolveNameBank({ 'Antidote++ (4)': 1, "Zulrah's scales": 20 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.17
+	},
+	{
+		name: 'Super combat potion (4)',
+		aliases: ['super combat potion (4)', 'super combat potion', 'super combat'],
+		id: itemID('Super combat potion (4)'),
+		level: 90,
+		xp: 150,
+		inputItems: resolveNameBank({
+			'Super attack(4)': 1,
+			'Super strength (4)': 1,
+			'Super defence (4)': 1,
+			Torstol: 1
+		}),
+		tickRate: 2,
```

Comment:
slightly increase tickRate to make the xp/hr reflect the wiki better since all other potions seems to reflect the wiki page.

Reply:
fixed